velero-plugin-for-gcp/volume_snapshotter.go

```diff
@@ -37,7 +37,8 @@ import (
 
 const (
 	zoneSeparator       = "__"
-	projectKey          = "project"
+	snapshotProjectKey  = "snapshotProject"
```

Comment:
i'm wondering if `volumeProject` would be a little more clear here, since this *has* to be the project that contains the volumes, it can't be anything else, and it only needs to be specified if it's different than the project containing the IAM account. What do you think?